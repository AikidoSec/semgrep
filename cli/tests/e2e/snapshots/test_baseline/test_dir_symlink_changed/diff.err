[33mBad settings format; /tmp/tmp0rbq3wjp will be overriden. Contents:
has_shown_metrics_notification: true


┌─────────────┐
│ Scan Status │
└─────────────┘
  Scanning 0 files tracked by git with 1 Code rule:
  Nothing to scan.
  Current version has 0 findings.

Skipping baseline scan, because there are no current findings.


┌──────────────┐
│ Scan Summary │
└──────────────┘
Some files were skipped or only partially analyzed.
  Scan was limited to files changed since baseline commit.

Ran 1 rule on 0 files: 0 findings.
