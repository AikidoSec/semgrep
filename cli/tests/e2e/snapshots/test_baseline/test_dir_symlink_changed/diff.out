None
EngineType.OSS
path to core /home/runner/work/semgrep/semgrep/cli/src/semgrep/bin/semgrep-core
